ha/util/conf_store.py

```diff
@@ -221,3 +227,24 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+
+    @staticmethod
+    def get_cvg_for_disk(node_id, disk_id):
+        """
+        Returns cvg id for given node_id and disk_id.
+        Args:
+            index(str): index of conf file
```

Comment:
We don't need this index args here, right ?

Reply:
Yes, it has become an obsolete statement, I will remove it,

Reply:
Done.